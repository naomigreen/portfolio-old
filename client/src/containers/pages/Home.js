import React from 'react';
import ReactMarkdown from 'react-markdown';
import styled from 'styled-components';
import { Image } from '../../components/Images/Images';
import { home } from '../../components/text/Text';

const Home = () => {
	return (
		<Intro>
			<Image
				src='https://naomi-assets.s3.eu-west-2.amazonaws.com/me.jpg'
				width='150px'
				radius='100%'
				padding='30px 0 10px'
			/>
			<Text source={home} escapeHtml={false} />
		</Intro>
	);
}

const Text = styled(ReactMarkdown)`
  padding: 20px;
  width: 500px;
  max-width: 90%;
  textAlign: center;
  margin: 0 auto;
  display: block;
`;
const Intro = styled.div`margin: 0 auto;`;

export default Home;