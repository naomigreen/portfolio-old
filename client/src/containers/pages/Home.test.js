import React from 'react';
import { render } from '@testing-library/react';
import Home from './Home';

describe('Home component', () => {
  const { asFragment } = render(<Home />);

  it('should render Home component', () => {
    expect(asFragment()).toMatchSnapshot();
  });


});
