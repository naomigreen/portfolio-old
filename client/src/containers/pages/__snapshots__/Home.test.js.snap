// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render Home component 1`] = `
<DocumentFragment>
  <div
    class="sc-AxhUy YVgRz"
  >
    <img
      alt=""
      class="sc-AxjAm fqTcMn"
      radius="100%"
      src="https://naomi-assets.s3.eu-west-2.amazonaws.com/me.jpg"
      width="150px"
    />
    <div
      class="sc-AxhCb kScNmO"
    >
      <p>
        Iâ€™m Naomi Prescod-Green, a Full Stack JavaScript Contractor and founder of Codes Green Ltd. 
        <br />
         
I have a Computer Science background and specialise in React, Vanilla and Node JavaScript. 
        <br />
         
I enjoy working on complex projects that test my skills and teach me new approaches to software 
development.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
